<launch>
    <arg name="file_name" default="20000" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find toy_map_viewer)/rviz/config.rviz"/>
    
    <node pkg="rf_tf_broadcaster" type="map_tf_broadcaster_node" name="map_tf_broadcaster_node" output="screen" />

    <node pkg="toy_map_viewer" type="map_loader_node" name="map_loader" output="screen">
        <param name="file_name" value="$(arg file_name)" type="str" /> 
    </node>
    
    <node pkg="toy_map_viewer" type="bin_map_viewer" name="map_viewer" output="screen">
        <param name="file_name" value="$(arg file_name)" type="str" /> 
    </node>
</launch>